packages/nbdime/src/diff/widget/metadata.ts

```diff
@@ -18,8 +22,13 @@ const ROOT_METADATA_CLASS = 'jp-Metadata-diff';
  * MetadataWidget for changes to Notebook-level metadata
  */
 export class MetadataDiffWidget extends Panel {
-  constructor(model: IStringDiffModel) {
+  // TODO improve typing hierarchy to avoid `Omit`
```

Comment:
Is this TODO for yourself, or for someone in the future? :)

Reply:
I don't think this comment is mine.

Reply:
I added it - I will remove it when working into the integration with `@jupyterlab/git`